css-pseudo-4/Overview.bs

```diff
@@ -879,6 +890,43 @@ Painting the Highlight</h3>
     css/css-pseudo/highlight-painting-003.html
     </wpt>
 
+<h4 id="paired-cascade">
+Highlight Colors</h4>
```

Comment:
This should be a subsection of “Default UA Styles”.

Reply:
i think you’re right, but only if we make it clearer for implementors that this is a used-value-time thing that doesn’t affect the computed value. imo it would be enough by linkifying “used” to used value again and not using the term “paired cascade”, despite its historical relevance as what we used in the note and in #6386

Reply:
done in e94418630480cf5c9ac640b207e446b33e890cc7